bin/i18n/sync-out.rb

```diff
@@ -23,11 +23,12 @@
 
```

Comment:
IMPORTANT: Don't forget to add the import line
`Dir[File.expand_path('../resources/**/*.rb', __FILE__)].sort.each {|file| require file}`

Reply:
Thank you!